clients/drcachesim/CMakeLists.txt

```diff
@@ -407,16 +404,24 @@ endif (X64)
 set(BUILD_CLIENTS_BIN clients/${INSTALL_BIN})
 set(BUILD_CLIENTS_LIB clients/${INSTALL_LIB})
 
-write_config_file(${CONFIG_INSTALL} ${INSTALL_CLIENTS_BIN} ${INSTALL_CLIENTS_LIB})
-write_config_file(${CONFIG_BUILD} ${BUILD_CLIENTS_BIN} ${BUILD_CLIENTS_LIB})
+if (DEBUG)
+  set(debugopt "TOOL_OP=-dr_debug")
```

Comment:
It seems nicer to keep all of the config file syntax inside write_config_file()

Reply:
Done